cmd/bricksllm/.env

```diff
@@ -0,0 +1,27 @@
+CONFIG_FILE_NAME=./config_local.json
+
+POSTGRESQL_HOSTS=localhost
+POSTGRESQL_DB_NAME=postgres
+POSTGRESQL_USERNAME=postgres
+POSTGRESQL_PASSWORD=postgres
+POSTGRESQL_SSL_MODE=disable
+POSTGRESQL_PORT=5432
+REDIS_HOSTS=localhost
```

Comment:
i would imagine that the values used here are for env defaults. if that is the case, you should match these default values with env defaults specified here: https://github.com/bricks-cloud/BricksLLM/blob/main/internal/config/config.go

Reply:
done.